test/functional/DDR_Test/src/j9vm/test/ddrext/junit/TestThread.java

```diff
@@ -265,21 +250,12 @@ private String getStackSlotsParamSet(String stackOutput,
 						els = token.split("=")[1].trim();
```

Comment:
This should be removed (as well as the declaration of `els`). The method could have been static before; please make it so now.

Reply:
The local `els` is still here, and the method comment for `getStackSlotsParamSet()` still refers to `needELS`.